src/asm3/wordprocessor.py

```diff
@@ -1602,7 +1602,12 @@ def event_tags(dbo, e):
         "EVENTCREATEDBY":       e["CREATEDBY"],
         "EVENTCREATEDDATE":     python2display(l, e["CREATEDDATE"]),
         "EVENTLASTCHANGEDBY":   e["LASTCHANGEDBY"],
-        "EVENTLASTCHANGEDDATE": python2display(l, e["LASTCHANGEDDATE"])
+        "EVENTLASTCHANGEDDATE": python2display(l, e["LASTCHANGEDDATE"]),
+        "EVENTOWNERNAME":       e["EVENTOWNERNAME"],
+        "EVENTADDRESS":         e["EVENTADDRESS"],
+        "EVENTTOWN":            e["EVENTCOUNTY"],
+        "EVENTPOSTCODE":        e["EVENTPOSTCODE"],
+        "EVENTCOUNTRY":         e["EVENTCOUNTRY"]
```

Comment:
Not really an issue, but I would have the audit fields for created/lastchanged last in the list